shared/actions/chat.js

```diff
@@ -764,6 +764,10 @@ function * _setupChatHandlers (): SagaGenerator<any, any> {
       dispatch({type: 'chat:inboxStale', payload: undefined})
     })
 
+    engine().setIncomingHandler('chat.1.NotifyChat.ChatTLFResolve', ({convID, resolveInfo: {newTLFName}}) => {
+      dispatch({type: 'chat:inboxStale', payload: undefined})
```

Comment:
yep, I first actually did just the relevant inbox, but it was a little more involved than this simple change